### PR TITLE
Add deployment tutorial to the readme.

### PR DESCRIPTION
**Pull request by fiveisprime**
Moved some contet from the readme to the wiki and added a short tutorial
for getting started with deployments from the extension.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,58 +2,41 @@
...
-* Upload and Download Blobs up to 4MB in size
-* Access Connection String and Primary Key
-* Open in Storage Explorer for memory or computationally heavy tasks, or for upload/download of large or non-text files.
+1. Download and install the [Azure Storage extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestorage) for Visual Studio Code
+2. Once complete, you'll see an Azure icon in the Activity Bar
+    > If your activity bar is hidden, you won't be able to access the extension. Show the Activity Bar by clicking View > Appearance > Show Activity Bar
+3. Sign in to your Azure Account by clicking Sign in to Azure...
+    >  If you don't already have an Azure Account, click "Create a Free Azure Account" for a free 30-day account with $200 in Azure credits to try out any combination of services.
 
-## Extension Settings
```
Comment (StephenWeatherford) [inline README.md]: Why did you remove extension settings?

File: README.md [hunk 2]
```diff
@@ -2,58 +2,41 @@
...
 
-The Azure Storage extension for VS Code lets you deploy static websites and browse Azure Blob Containers, File Shares, Tables, and Queues. Follow [this tutorial](https://code.visualstudio.com/tutorials/static-website/getting-started) to deploy your web apps to Azure Storage from VS Code.
+Azure Storage is a Microsoft-managed service providing cloud storage that is highly available, secure, durable, scalable, and redundant. Use the extension to deploy static websites and Single Page Apps (SPAs) and browse Azure Blob Containers, File Shares, Tables, and Queues.
 
-## Features
+**Visit the [wiki](https://github.com/Microsoft/vscode-azurestorage/wiki) for more additional information about the extension.**
```
Comment (StephenWeatherford) [inline README.md]: How about putting this in a section header of its own?  I never would have noticed this.

Comment (StephenWeatherford) [inline README.md]: "more additional"?  :-)

File: README.md (hunk 1 above)
Comment (StephenWeatherford) [reply README.md]: Personally I'm skeptical of having this info in a separate wiki.  A lot of people already don't find the readme.

Comment (fiveisprime) [reply README.md]: Trying to keep the readme as concise as possible.

Comment (fiveisprime) [reply README.md]: How aren't people finding the readme? Why do you say they aren't finding it? The settings aren't critical to using the extension so I moved it to the wiki.

File: README.md (hunk 2 above)
Comment (fiveisprime) [reply README.md]: Is the bold not good enough?

Comment (StephenWeatherford) [reply README.md]: It helps a bit, but if I'm skipping the file, I'm looking at headings.
image

Comment (fiveisprime) [reply README.md]: What if it's part of one of the other sections? It feels strange to have a header for a single, 10 word sentence. It would fit under the settings section.

Comment (fiveisprime) [reply README.md]: Ping
